docs/01-prerequisites.md

```diff
@@ -95,23 +93,7 @@ The key's randomart image is:
 +----[SHA256]-----+
 ```
 
-Establish the connection between the kcli and the local libvirt.
-
-```
-kcli create host kvm -H 127.0.0.1
-```
-
-Verify
-
-```
-+-----------+------+---------+---------+
-| Client    | Type | Enabled | Current |
-+-----------+------+---------+---------+
-| localhost | kvm  |   True  |    X    |
-+-----------+------+---------+---------+
-```
-
-Next step is create a pool where the cloud images are donwloaded and where the vms are going to be placed in the baremetal server. In this case, we use the defacto libvirt image path:
+Next step is to create a libvirt pool where the cloud images are donwloaded and where the vms are going to be placed in the baremetal server:
 
```

Comment:
I would leave it as is in order to let the reader know that there is a connection established between kcli and the local libvirt daemon. Don't you?

Reply:
i dont like to state things this way because there s no connection of any kind created. Kcli just defaults to local libvirt daemon if no additional configuration is provided. the kcli create host command, when targeting local hypervisor has only one benefit, to generate a config file a user can expand on afterwards